src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService validationUser = new UserService();
+        boolean result = false;
+        if (validationUser.findByEmail(email) != null
+                && validationUser.findByEmail(email).getPassword().equals(password)) {
```

Comment:
do not call `findByEmail` 2 times

Reply:
I don't know how could  I get access to password field beside calling findByEmail.